_site/index.html

```diff
@@ -0,0 +1,214 @@
+            href="/lang/en">
+            english (en)
+          </a>
+        </li>
+      
+    </ol>
+    <ol class="nav">
+      
+      
+        
+        
+        
+          <li class="version version-1.0.0-beta">
+            <a href="/spec/v1.0.0-beta.html">
+              1.0.0-beta
+            </a>
+          </li>
+        
+      
+    </ol>
+    <div id="spec">
+     <h1 id="conventional-commits-100-beta">Conventional Commits 1.0.0-beta</h1>
+
+<h2 id="summary">Summary</h2>
+
+<p>As an open-source maintainer, squash feature branches onto <code class="highlighter-rouge">master</code> and write
+a standardized commit message while doing so.</p>
+
+<p>The commit message should be structured as follows:</p>
+
+<hr />
+
+<div class="highlighter-rouge"><pre class="highlight"><code>&lt;type&gt;[optional scope]: &lt;description&gt;
+
+[optional body]
+
+[optional footer]
+</code></pre>
+</div>
+<hr />
+
+<p><br />
+The commit contains the following structural elements, to communicate intent to the
+consumers of your library:</p>
+
+<ol>
+  <li><strong>fix:</strong> a commit of the <em>type</em> <code class="highlighter-rouge">fix</code> patches a bug in your codebase (this correlates with <a href="http://semver.org/#summary"><code class="highlighter-rouge">PATCH</code></a> in semantic versioning).</li>
+  <li><strong>feat:</strong> a commit of the <em>type</em> <code class="highlighter-rouge">feat</code> introduces a new feature to the codebase (this correlates
+  with <a href="http://semver.org/#summary"><code class="highlighter-rouge">MINOR</code></a> in semantic versioning).</li>
+  <li><strong>BREAKING CHANGE:</strong> a commit that has the text <code class="highlighter-rouge">BREAKING CHANGE:</code> at the beginning of its optional body or footer section introduces a breaking API change (correlating with <a href="http://semver.org/#summary"><code class="highlighter-rouge">Major</code></a> in semantic versioning). A breaking change can be
+  part of either a <code class="highlighter-rouge">fix:</code> or <code class="highlighter-rouge">feat:</code> <em>type</em> commit.</li>
+</ol>
+
+<p><br />
+A scope may be provided to a commit’s type, to provide additional contextual information and
+is contained within parenthesis, e.g., <code class="highlighter-rouge">feat(parser): adds ability to parse arrays</code>.</p>
+
+<p>Commit <em>types</em> other than <code class="highlighter-rouge">fix:</code> and <code class="highlighter-rouge">feat:</code> are allowed, for example <a href="https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format">the angular convention</a> recommends <code class="highlighter-rouge">docs:</code>, <code class="highlighter-rouge">style:</code>, <code class="highlighter-rouge">refactor:</code>, <code class="highlighter-rouge">perf:</code>, <code class="highlighter-rouge">test:</code>, <code class="highlighter-rouge">chore:</code>, but these tags are
```

Comment:
Same here. 
Also should we use "Angular" instead of "angular" throughout the site?